### PR TITLE
web stuff

### PR DESCRIPTION
**Pull request by antonbobkov**

### CODE REVIEW

File: web_files/www/form.html [hunk 1]
```diff
@@ -2,10 +2,13 @@
...
 <body>
 
-<h2>The method Attribute</h2>
+<a href="/erl/potato_cluster_web_server:get_system_info">system info</a>
+
+<br>
+
 <p>This form will be submitted using the POST method:</p>
 
-<form action="/erl/hello_web:service" target="_blank" method="POST">
```
Comment (pdlla) [inline web_files/www/form.html]: +1 to rename

File: src/potato_cluster_web_server.erl [hunk 2]
```diff
@@ -53,8 +57,50 @@ stop(Reference) ->
...
+
+    % Erlang memory usage
+    ErlMem = erlang:memory(),
+    ErlMemTotal = erlang:memory(total) / 1000000,
+
+    % extenal os memory usage
+    SysMem = os:cmd("free -mt"),
+
+    % docker memory usage
+    {DockerByteMem, _} = string:to_integer(os:cmd("cat /sys/fs/cgroup/memory/memory.usage_in_bytes")),
```
Comment (antonbobkov) [inline src/potato_cluster_web_server.erl]: "cat" as in :scream_cat:

**Comment by antonbobkov** [comment]
OMG, i've managed to shut up codecov

so good

File: src/potato_cluster_web_server.erl (hunk 2 above)
Comment (pdlla) [inline src/potato_cluster_web_server.erl]: :o

**Comment by antonbobkov** [comment]
lol, nvm

**Comment by pdlla** [comment]
🙀

**Comment by antonbobkov** [comment]
gotta get that yaml right

**Comment by pdlla** [comment]
hey, I saw that. That's cheating  :o.

**Comment by antonbobkov** [comment]
just to see if it gets reposted!

**Comment by antonbobkov** [comment]
fingers crossed

**Comment by antonbobkov** [comment]
YAY

**Pull request merged**